build.sbt

```diff
@@ -1,120 +1,110 @@
 
-excludeDependencies ++= Seq(
-  "org.consensusresearch" %% "scrypto"
-)
+lazy val common = (project in file("conseil-common"))
+  .settings(
+    name := "conseil-common",
+    libraryDependencies ++= Dependencies.conseilCommonInclude,
+    excludeDependencies ++= Dependencies.conseilCommonExclude,
+    coverageExcludedPackages := Seq(
+          "<empty>",
+          "tech.cryptonomic.conseil.common.io.*",
+          "tech.cryptonomic.conseil.common.tezos.Tables",
+          "tech.cryptonomic.conseil.common.config.Security"
+        ).mkString(";")
+  )
+  .settings(Defaults.itSettings)
+  .settings(
+    scalacOptions += "-P:silencer:pathFilters=common/src/main/scala/tech/cryptonomic/conseil/common/tezos/Tables.scala"
+  )
+  .configs(IntegrationTest)
+  .enableBuildInfo()
+  .disableAssembly()
 
-assemblyOutputPath in assembly := file("/tmp/conseil.jar")
+lazy val api = (project in file("conseil-api"))
+  .settings(
+    name := "conseil-api",
+    mainClass := Some("tech.cryptonomic.conseil.api.Conseil"),
+    libraryDependencies ++= Dependencies.conseilApiInclude,
+    coverageExcludedPackages := Seq(
+          "<empty>",
+          ".*\\.Conseil",
+          ".*\\.ConseilAppConfig"
+        ).mkString(";")
+  )
+  .settings(Defaults.itSettings)
+  .configs(IntegrationTest)
+  .addRunCommand(
+    description = "A run application Task.",
+    javaExtras = Seq("-Xms1024M", "-Xmx8192M", "-Xss1M", "-XX:+CMSClassUnloadingEnabled")
+  )
+  .enableAssembly()
+  .dependsOn(common % "compile->test")
 
-scalacOptions ++= ScalacOptions.common
-scalacOptions += "-P:silencer:pathFilters=src/main/scala/tech/cryptonomic/conseil/tezos/Tables.scala"
-import complete.DefaultParsers._
+lazy val lorre = (project in file("conseil-lorre"))
+  .settings(
+    name := "conseil-lorre",
+    mainClass := Some("tech.cryptonomic.conseil.lorre.Lorre"),
+    libraryDependencies ++= Dependencies.conseilLorreInclude,
+    coverageExcludedPackages := Seq(
+          "<empty>",
+          ".*\\.Lorre",
+          ".*\\.LorreAppConfig"
+        ).mkString(";")
+  )
+  .settings(Defaults.itSettings)
+  .configs(IntegrationTest)
+  .addRunCommand(
+    description = "A run application Task.",
+    javaExtras = Seq("-Xmx512M", "-Xss1M", "-XX:+CMSClassUnloadingEnabled")
+  )
+  .enableAssembly()
+  .dependsOn(common)
 
-lazy val runConseil = inputKey[Unit]("A conseil run task.")
-fork in runConseil := true
-javaOptions in runConseil ++= Seq("-Xms1024M", "-Xmx8192M", "-Xss1M", "-XX:+CMSClassUnloadingEnabled")
-runConseil := Def.inputTaskDyn {
-  val args = spaceDelimited("").parsed
-  runInputTask(Runtime, "tech.cryptonomic.conseil.Conseil", args: _*).toTask("")
-}.evaluated
+lazy val lorreSchema = (project in file("conseil-lorre-schema"))
+  .settings(
+    name := "conseil-lorre-schema",
+    mainClass := Some("tech.cryptonomic.conseil.lorre.schema.GenSchema")
+  )
+  .addRunCommand(description = "A run schema generating Task.")
+  .enableAssembly()
```

Comment:
I don't think we need assembly for that

Reply:
Ok, I removed that, expect the change at GitHub today.